demo_scenario.py

```diff
@@ -31,22 +31,26 @@ def set_up_scenario(
                 "V_cap"],
             digital_signals = [])
 
+        CAPTURE_DURATION = 0.1
```

Comment:
is this used as an offset?

Reply:
This specifies the time duration of the high resolution data capture (i.e. recording the data on each simulation time step, which will rapidly generate large amounts of data and hence the brief capture window)